back-end/controllers/UserController.js

```diff
@@ -23,6 +23,15 @@ const getUser = async (req, res) => {
 const updateUserById = async (req, res) => {
   try {
     const { id } = req.params;
+    const existEmail = await findByEmail(req.body.email);
+    if (existEmail) {
+      const body = {
+        fullName: req.body.fullName,
+        dateOfBirth: req.body.dateOfBirth,
+      };
+      await updateByID(id, body);
```

Comment:
Are you sure that when we have existEmail we need to do update user?

Reply:
I thought should update what pass validation

Reply:
Okay. Why you update and return 400 fail status code?